packages/menu-react/documentation/MenuToggleSwitchExample.tsx

```diff
@@ -0,0 +1,119 @@
+import { type ColorScheme } from "@fremtind/jkl-core";
+import { IconButton } from "@fremtind/jkl-icon-button-react";
+import { DotsIcon } from "@fremtind/jkl-icons-react";
+import { useBrowserPreferences } from "@fremtind/jkl-react-hooks";
+import React, { useCallback, useRef, useState, type FC } from "react";
+import { CodeExample, ExampleComponentProps, ExampleKnobsProps } from "../../../doc-utils";
+import { Tooltip, TooltipContent, TooltipTrigger } from "../../tooltip-react/build";
```

Comment:
Denne må fikses. Tror det riktigste blir

```suggestion
import { Tooltip, TooltipContent, TooltipTrigger } from "../../tooltip-react/src";
```